util/is-inverse-benchmark-record.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {{}} record
+ * @return {boolean}
+ */
+var isInverseBenchmarkRecord = function isInverseBenchmarkRecord(record) {
```

Comment:
yes -- node is smart enough to use the name of the var in stack traces, so there is no longer any reason to name the function twice. either name the var, or the function